FreeRTOS-Plus/Demo/MQTT_Mutual_Auth_Qemu_mps2/CMSIS/CMSDK_CM3.h

```diff
@@ -0,0 +1,723 @@
+/* MPS2 CMSIS Library
```

Comment:
Can these files be shared with the other MPS2 demos?

Reply:
Yes, but it seems that it is included in each of the demos at the moment